src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -838,64 +759,61 @@ def get_existing_app_info(self) -> Optional[dict]:
         Check for an existing application object by the same name as in project definition, in account.
         It executes a 'show applications like' query and returns the result as single row, if one exists.
         """
-        sql_executor = get_sql_executor()
-        with sql_executor.use_role(self.role):
-            return sql_executor.show_specific_object(
-                "applications", self.name, name_col=NAME_COL
-            )
+        return get_snowflake_facade().get_existing_app_info(self.name, self.role)
```

Comment:
i am confused, why do we use facade version sometimes, and sometimes use the application one (self.get_existing_app_info)?

Reply:
My mistake, I forgot to refactor the other places where we call that method, I just replaced them all now